### ISSUE TITLE
Falling back to eth_sync api

### ISSUE THREAD

**Issue by agatsoh** [raiden-network/raiden#5042]
## Problem Definition

While starting raiden get this 
```
Cannot use https://goerli.etherscan.io/api?module=proxy&action=eth_blockNumber. Request failed
Falling back to eth_sync api.
```
These logs are seen both while using the brainbot eth nodes as well as on infura.

Though it has absolutely no effect and raiden seems to work fine with respect to all its functions.

**Comment by christianbrb** [comment]
@rakanalh Would that have prevented the other issues like #5044 ? Did eth_sync did not discover the issues the Geth node had?

**Comment by rakanalh** [comment]
No this is totally unrelated. The above was just a warning saying we could not check that the eth node was in sync. We do this initially by requesting the latest block number from etherscan.
The request to etherscan failed because they recently started serving through CloudFlare and `user-agent` seemed to be necessary for the request to go through.

But even with the fix, our request to etherscan could fail and we fallback to using a different method.

Bottom line: the above happens during startup sync check and unrelated to nonce issues or anything else.

**Comment by rakanalh** [comment]
#5044 documents what was wrong
